package.json

```diff
@@ -28,9 +28,7 @@
     }
   },
   "babel": {
-    "plugins": [
-      "inline-react-svg"
-    ],
+    "plugins": [],
```

Comment:
We will need this one. If you check some components like [ListItem](https://taskforce-fe-compon-git-fork-victorbojica-fix-export-com-e148c8.code4ro.now.sh/?path=/story/listitem--with-inactive-state), it breaks. This module is not doing the same thing. `@rollup/plugin-image` takes care of exporting/importing and `inline-react-svg` makes sure when we use `SVGs` inline they are valid elements. We need these so we can change colors of those svgs, for example. Can you please revert that commit? 🙌

Reply:
You are right, my bad, sorry! :)